plugins/soundcheck-backend-module-branch/src/branchcount.ts

```diff
@@ -96,6 +96,20 @@ export class BranchCountFactCollector implements FactCollector {
     }));
   }
 
+  /**
+   * Collect facts for a set of entities. When called by the Soundcheck
+   * backend, this method typically receives a single entity at a time,
+   * but this behavior is not guaranteed. The collector should return
+   * a list of facts for each given entity according to the given
+   * parameters. If no fact references are provided, the collector should
```

Comment:
```
If no fact references are provided, the collector should return all facts that it can collect for the given entity.
```

That's an interesting thing I did not know